example/pubspec.yaml

```diff
@@ -25,7 +25,7 @@ dependencies:
   flutter:
     sdk: flutter
   file_picker: ^1.8.0+1
-  amplify_core: ^0.0.1-dev.4
+  amplify_flutter: ^0.0.1-dev.4
```

Comment:
It feels a little weird to roll out a "new" package with this version, but I suppose if we are keeping all these numbers in sync it makes sense.